tests/test_project_structure.py

```diff
@@ -56,18 +56,23 @@ def test_providers_modules_should_have_tests(self):
         """
         Assert every module in /airflow/providers has a corresponding test_ file in tests/airflow/providers.
         """
+        # Deprecated modules that don't have corresponded test
+        expected_missing_providers_modules = {('airflow/providers/amazon/aws/hooks/aws_dynamodb.py',
+                                               'tests/providers/amazon/aws/hooks/test_aws_dynamodb.py')}
+
         # TODO: Should we extend this test to cover other directories?
-        expected_test_files = glob.glob(f"{ROOT_FOLDER}/airflow/providers/**/*.py", recursive=True)
+        modules_files = glob.glob(f"{ROOT_FOLDER}/airflow/providers/**/*.py", recursive=True)
+
         # Make path relative
-        expected_test_files = (os.path.relpath(f, ROOT_FOLDER) for f in expected_test_files)
+        modules_files = (os.path.relpath(f, ROOT_FOLDER) for f in modules_files)
         # Exclude example_dags
-        expected_test_files = (f for f in expected_test_files if "/example_dags/" not in f)
+        modules_files = (f for f in modules_files if "/example_dags/" not in f)
         # Exclude __init__.py
-        expected_test_files = (f for f in expected_test_files if not f.endswith("__init__.py"))
+        modules_files = (f for f in modules_files if not f.endswith("__init__.py"))
```

Comment:
I think the previous name was more informative

Reply:
I changed the name to `modules_files` because at this point these are not yet the files in the `test/` path.
the `modules_files` are needed for the 2nd new subtest. Thus we need to distinguish between the test modules to the actual modules. WDYT?

Reply:
I'm ok with that